proto/cosmos/staking/v1beta1/genesis.proto

```diff
@@ -51,3 +117,24 @@ message LastValidatorPower {
   // power defines the power of the validator.
   int64 power = 2;
 }
+
+// Development plan
+// — Implement changes for Msgs flow
+// — — There are 6 queues to be newly added for epoching process and should write a common utility to manage queues by considering existing staking module codebase
+// — — Implement Msgs handling one by one (implement instant change fields, queue actions for epoching)
+// — Implement logic flow after queue action
+// — Implement CLI commands for querying
+// — — BufferedValidators
+// — — BufferedMsgCreateValidatorQueue, BufferedMsgEditValidatorQueue
+// — — BufferedMsgUnjailQueue, BufferedMsgDelegateQueue, BufferedMsgRedelegationQueue, BufferedMsgUndelegateQueue
```

Comment:
Should we combine BufferedMsgCreateValidatorQueue, BufferedMsgEditValidatorQueue, BufferedMsgDelegateQueue, BufferedMsgRedelegationQueue, and BufferedMsgUndelegateQueue into a single queue?

Only BufferedMsgUnjailQueue **needs** to be a separate queue, because its in `x/slashing`, not `x/staking`.

Reply:
I think it would be good to merge to make things easier.